cycode/cli/files_collector/sca/go/restore_go_dependencies.py

```diff
@@ -0,0 +1,31 @@
+import os
+from typing import List
+
+import click
+
+from cycode.cli.files_collector.sca.base_restore_dependencies import BaseRestoreDependencies
+from cycode.cli.models import Document
+
+GO_PROJECT_FILE_EXTENSIONS = ['.mod']
+GO_RESTORE_FILE_NAME = 'go.sum'
+BUILD_GO_FILE_NAME = 'go.mod'
+
+
+class RestoreGoDependencies(BaseRestoreDependencies):
+    def __init__(self, context: click.Context, is_git_diff: bool, command_timeout: int) -> None:
+        super().__init__(context, is_git_diff, command_timeout, True)
+
+    def is_project(self, document: Document) -> bool:
+        return any(document.path.endswith(ext) for ext in GO_PROJECT_FILE_EXTENSIONS)
+
+    def get_command(self, manifest_file_path: str) -> List[str]:
+        return ['cd', self.prepare_tree_file_path_for_command(manifest_file_path), '&&', 'go', 'list', '-m', '-json']
```

Comment:
are you sure that this command will be executed on Windows normally? `cd` and `&&` worry me

Reply:
they works on powershell as well

Reply:
no, the CD and && works properly on cmd as well